docs/fsharp-cheatsheet.md

```diff
@@ -125,33 +138,56 @@ See [Literals (MS Learn)](https://learn.microsoft.com/en-us/dotnet/fsharp/langua
 
 ## Functions
 
-The `let` keyword also defines named functions.
+Use the `let` keyword to define named functions.
 
     let negate x = x * -1
     let square x = x * x
-    let print x = printfn "The number is: %d" x
+    let print x = printfn $"The number is: {x}"
+
+### Pipe and Composition Operators
 
-### Pipe and composition operators
 Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to improve readability especially in unit testing:
 
     let ``square, negate, then print`` x =
         x |> square |> negate |> print
 
-This operator can assist the F# type checker by providing type information before use:
-
-    let sumOfLengths (xs : string []) =
-        xs
-        |> Array.map (fun s -> s.Length)
-        |> Array.sum
-
 Composition operator `>>` is used to compose functions:
 
-    let squareNegateThenPrint' =
+    let squareNegateThenPrint =
         square >> negate >> print
 
-<div id="recursive-functions"></div>
+### Signatures and Explicit Typing
+
+Function signatures are useful for quickly learning the input and output of functions. The last type is the return type and all preceding types are the input types.
+
+    int -> string                       // this defines a function that receives an integer; returns a string
+    int -> int -> string                // two integer inputs; returns a string
+    unit -> string                      // no inputs; returns a string
```

Comment:
it requires a unit input though - kinda need to convey that an individual arg can be unit/primitive/tuple